### ISSUE TITLE
Players assigned to wrong teams

### ISSUE THREAD

**Issue by jack89roberts** [alan-turing-institute/AIrsenal#95]
The most frequent data issue that's come up from working on sanity checks in #62 is players being associated with the wrong team (presumably after moving clubs within the premier league). We should look at how the PlayerAttributes table is being populated and make sure we're always capturing player moves, both for previous seasons and the current season.

**Comment by jack89roberts** [comment]
A few examples of issues in fixtures near the end of last season:
```
1819 GW22 EVE vs BOU (id 162): Oumar Niasse in player_scores but labelled as playing for CAR.
1819 GW21 MCI vs LIV (id 171): Nathaniel Clyne in player_scores but labelled as playing for BOU.
1819 GW21 MCI vs LIV (id 171): Dominic Solanke in player_scores but labelled as playing for BOU.
1819 GW21 WOL vs CRY (id 172): Jason Puncheon in player_scores but labelled as playing for HUD.
```

**Comment by jack89roberts** [comment]
This is fixed as part of #117
